fdbserver/ClusterController.actor.cpp

```diff
@@ -802,6 +802,11 @@ class ClusterControllerData {
 			return false;
 		}
 
+		// Do not trigger better master exists if the cluster controller is excluded, since the master will change anyways once the cluster controller is moved
+		if(id_worker[clusterControllerProcessId].priorityInfo.isExcluded) {
```

Comment:
The cluster controller cannot recruit a master before clusterControllerProcessId is set, and once it is set it (and its worker in id_worker) can never be removed, Therefore all the places that use it blindly are safe because all of that code (including this one) can only ever be run after the first master is recruited